src/main/resources/init_db.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE `library_db` /*!40100 DEFAULT CHARACTER SET utf8 */ /*!80016 DEFAULT ENCRYPTION='N' */;
+
+CREATE TABLE manufacturers (
+  id bigint NOT NULL AUTO_INCREMENT,
+  name varchar(255) DEFAULT NULL,
+  country varchar(255) DEFAULT NULL,
+  is_deleted tinyint NOT NULL DEFAULT '0',
+  PRIMARY manufacturers KEY (id)
```

Comment:
```suggestion
CREATE TABLE `manufacturers` (
  `id` bigint NOT NULL AUTO_INCREMENT,
  `name` varchar(255) DEFAULT NULL,
  `country` varchar(255) DEFAULT NULL,
  `is_deleted` tinyint NOT NULL DEFAULT '0',
  PRIMARY KEY (`id`)
```
or does it work as it is at the moment?

Reply:
it working